config/sync/ultimate_cron.job.layout_builder_cron.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 7c618462-dfc2-4ba0-8eb0-608c1c5273ff
```

Comment:
Does layout builder need cron? I'm guessing this is auto-exported but wondering what things you'd want to schedule with it?

Reply:
It adds an entry into ultimate cron. Complete guess but it's possibly in there to do routine cleanup of displays/layouts etc.